lib/PipelineC/PipelineC.cpp

```diff
@@ -617,6 +617,24 @@ const char *rp_manager_get_global_name(rp_manager *manager, int index) {
   return nullptr;
 }
 
+static void errorToStrings(llvm::Error Error, std::vector<rp_error> &Strings) {
+  // clang-format off
+  llvm::handleAllErrors(std::move(Error),
+    [&](const revng::DiffError &Error) {
```

Comment:
Why `DiffError`? We did all these tricks to be able to use a base class here, don't we?

Reply:
yeah, this was an example, we can handle revngErrorBase directly if that looks correct to you

Reply:
Yes! This kinda is point of all these castles `:)`